package.json

```diff
@@ -64,8 +64,8 @@
     "manage-roles:polygon:mainnet": "npx hardhat manage-roles --network polygon --env prod >> logs/polygon.manage.roles.txt",
     "manage-roles:local": "npx hardhat manage-roles --network localhost >> logs/localhost.manage.roles.txt",
     "manage-roles:test": "npx hardhat manage-roles --network test --env test >> logs/test.manage.roles.txt",
-    "natspec-interface-id": "npx hardhat compile && node scripts/util/natspec-interfaceid.js",
```

Comment:
Maybe I'm missing something, but it seems that `natspec-interfaceid.js` does not compile the contracts and just takes whatever is in the artefacts (which might be different from the latest contracts).
In GHA that's not an issue (since they are compiled just before), but locally we can't assume that user will call compile before calling this script.

Reply:
you are right. At first, I thought we were compiling in both CI and pre-commit hook before we called `natspec-interface-id`, so I made this change. But we don't actually compile in pre-commit hook. And also, I can't assume everyone is using the hook for fixing interface ids as I do 😅